### ISSUE TITLE
Initial window size too small on hi-res desktops

### ISSUE THREAD

**Issue by spatialdude** [Esri/Trek2There#63]
Windows size too small on Surface Book

Use this to fix - 

```qml
App {
    id: app

    readonly property real windowScaleFactor: !(Qt.platform.os === "windows" || Qt.platform.os === "unix" || Qt.platform.os === "linux") ? 1 : AppFramework.displayScaleFactor

    width: 1000 * windowScaleFactor
    height: 600 * windowScaleFactor
}
```

**Comment by defiantgoat** [comment]
@marikavertzonis This has been fixed with v 2.0.302

**Comment by marikavertzonis** [comment]
tested 2.0.302 on surface book (hi res) and surface pro (not as high res) - on both - the app occupies good proportion of the screen.
